packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-navigation-menu-content.js

```diff
@@ -51,6 +54,7 @@ function getBlocksOfTypeFromBlocks( targetBlockType, blocks ) {
 
 export default function useNavigationMenuContent( postType, postId ) {
 	const { record } = useEditedEntityRecord( postType, postId );
+	const selectedCoreStore = useSelect( coreStore );
```

Comment:
```suggestion
	const { getNavigationFallbackId } = useSelect( unlock(coreStore) );
```

I think we can stick with the patterns used elsewhere in the codebase here to make it clearer.